Obsidian/Net/Packets/Play/Clientbound/SpawnExperienceOrb.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Obsidian.API;
+using Obsidian.Entities;
+using Obsidian.Serialization.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Obsidian.Net.Packets.Play.Clientbound
+{
+    public partial class SpawnExperienceOrb : IPacket
+    {
+        [Field(0), VarLength]
+        public int EntityId { get; private set; }
+
+        [Field(1), Absolute]
+        public PositionF Position { get; private set; }
+
+        [Field(2)]
+        public short Count { get; private set; }
+
+        public int Id => 0x01;
+
+
```

Comment:
There should be an empty constructor (at least a private one). I don't think the project will even compile without it. (generators)